cmd/erigon/main.go

```diff
@@ -35,6 +38,14 @@ func runErigon(cliCtx *cli.Context) {
 	logger := log.New()
 	// initializing the node and providing the current git commit there
 	logger.Info("Build info", "git_branch", params.GitBranch, "git_tag", params.GitTag, "git_commit", params.GitCommit)
+
+	yamlFilePath := cliCtx.GlobalString(utils.ConfigFileFlag.Name)
+	if yamlFilePath != "" {
```

Comment:
So if `--config-file`, it does not overwrite other flags that are passed in of the same kind?
I ask because if `--config-file` is passed and `yamlFilePath` is empty, the code goes on to check other set flags

Reply:
--config-file is the path to the yamlFilePath, so if we get a yamlFilePath we set up all the flags inside of the yaml file, but we "overwrite" the yaml flags if they were set by hand in the cli.